plugins/kubernetes/app/views/samson_kubernetes/_deploy_changeset_tab_body.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @stage.kubernetes? %>
+  <div class="tab-pane" id="manifests">
+    <!-- using an iframe to lazy-load the manifest preview as it is potentially expensive / slow -->
```

Comment:
can we make this actually lazy-load (instead of in async parallel) ? ... not sure how the other tabs work

Reply:
I thought you hated JS? The other tabs are rendered inline, and this comes from an XHR once you select your reference and click preview.